bundles/org.openhab.ui/web/src/assets/definitions/widgets/plan/index.js

```diff
@@ -27,7 +27,7 @@ export const OhPlanMarkerDefinition = () => new WidgetDefinition('oh-plan-marker
     pi('item', 'Item', 'The item whose state to display on this marker')
   ])
   .paramGroup(pg('icon', 'Icon'), [
-    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/docs/configuration/iconsets/classic/">openHAB icon</a>) or <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>)'),
+    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/link/icons">openHAB icon</a>) or <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>)'),
```

Comment:
When I try do open this page I get a 404 Error. This link doesn't seem to work.
(Link I tested: https://www.openhab.org/link/icons/)